photosynthesis_metrics/vif.py

```diff
@@ -0,0 +1,138 @@
+r"""Implemetation of Visual Information Fidelity metric
+Code is based on MATLAB version for computations in pixel domain
+https://live.ece.utexas.edu/research/Quality/VIF.htm
+
+References:
+    https://ieeexplore.ieee.org/abstract/document/1576816/
+"""
+import torch
+import numpy as np
+from torch.nn.modules.loss import _Loss
+import torch.nn.functional as F
+
+from photosynthesis_metrics.utils import _adjust_dimensions, _validate_input
+
+
+def _gaussian_kernel2d(kernel_size: int = 5, sigma: float = 2.0) -> torch.Tensor:
```

Comment:
We can open an issue, but for now I would suggest to leave it like that. In SSIM 1-D kernels are used, and here they are 2-D. So there are not exactly the same.

Reply:
The ideas are actually the same, in case of SSIM we apply 1d kernel for X and Y, while here we apply 2d kernel. 
Nevertheless, I agree that we could consider it as additional issue/feature.